src/Qdrant.Client/QdrantClient.cs

```diff
@@ -3135,6 +3175,8 @@ public async Task<IReadOnlyList<AliasDescription>> ListAliasesAsync(Cancellation
 		string groupBy,
 		IReadOnlyList<PointId> positive,
 		IReadOnlyList<PointId>? negative = null,
+		ReadOnlyMemory<Vector>? positive_vectors = null,
```

Comment:
Codestyle `positiveVectors` and `negativeVectors`

Reply:
https://github.com/qdrant/qdrant-dotnet/pull/42/commits/2141e763ba18f9d237fb68a1ce0bee092834b750